isce2_topsapp/__main__.py

```diff
@@ -58,13 +62,13 @@ def localize_data(
         # For ionospheric correction computation
         if water_mask_flag:
             out_water_mask = download_water_mask(
-                out_slc["extent"],  water_name='SWBD')
+                out_slc["extent"],  water_name="SWBD")
 
         out_aux_cal = download_aux_cal()
 
-    out = {'reference_scenes': reference_scenes,
-           'secondary_scenes': secondary_scenes,
-           'frame_id': frame_id,
+    out = {"reference_scenes": reference_scenes,
+           "secondary_scenes": secondary_scenes,
+           "frame_id": frame_id,
```

Comment:
Ha - it's funny I am more a fan of `'`. I guess pycharm differentiates string keys and string values?

Reply:
@cmarshak we typically prefer single quotes as well. I suspect @forrestfwilliams was trying to match the existing style in `__main__.py`, which is heavier on the double quotes side